### DISCUSSION TITLE
Show commands to be learnt in the level selection panel

### DISCUSSION THREAD

**Discussion by Ruoye36** [pcottle/learnGitBranching#1039] [Ideas]
First, thank you so much for this fantastic tool, the best I have used to learn git!
Just wondering if, for each level, the commands to be learned in this level can be specified in the level selection panel? For now, it only shows the level title as shown in the green circle.
![Screenshot 2022-12-11 010444](https://user-images.githubusercontent.com/91307609/206895321-c2fa2155-7316-46f0-ba87-73c9352b9fd0.png)
Or could there be a search bar to input a specific command to look for the related levels? In this way, an unfamiliar command can be quickly found and learned.

**Comment by pcottle** [comment]
Great suggestion, but its a little tricky to actually put this into practice. There isn't a one to one mapping of "this lesson teaches you this command." Some commands are spread out over multiple lessons (since there are so many concepts to cover).

I think if you're looking for specific reference for a specific command, I would try looking online for documentation. Or just ask ChatGPT :P
